libs/sdk-ui-dashboard/src/model/commands/layout.ts

```diff
@@ -15,8 +25,13 @@ export interface AddLayoutSectionPayload {
      * @remarks
      * Index is zero-based and for convenience index -1 means place new section at the end. 0 means place new
      * section at the beginning. Both 0 and -1 and can be used when inserting the first section into and empty layout.
+     *
+     * @remarks
+     * {@link RelativeIndex} support will be removed in the next major SDK version. Use {@link ILayoutSectionPath} instead.
+     *
+     * TODO LX-648: Remove RelativeIndex type in the next major version.
```

Comment:
Commands are extended to support new addressing information which is optional for now to have backward compatibility. If provided, new logic in handlers is used.

Some handlers support old and new type on the original payload prop as on the command below.

TypeScript function override mechanism is used to have original command handlers as well as the new ones and user do not need to enter multiple undefined function parameters and loses for example correlationId information.

Each command have TODOs marked with Jira ticket that we need to address with the next major SDK release, after the feature productization, to cleanup this interface from all this legacy balast.